pepper-apis/src/main/java/org/broadinstitute/ddp/route/SendEmailRoute.java

```diff
@@ -49,10 +51,10 @@ public SendEmailRoute(WorkflowService workflowService) {
 
     @Override
     public Object handle(Request request, Response response, SendEmailPayload payload) throws Exception {
+        DDPAuth ddpAuth = RouteUtil.getDDPAuth(request);
         String email = payload.getEmail();
-
         String studyGuid = request.params(RouteConstants.PathParam.STUDY_GUID);
-
+        String operatorGuid = ddpAuth.getOperator();
```

Comment:
For inserting a new activity instance, we need an `operatorGuid`. `SendEmailRoute` is a bit special because it's called in the context of a study rather than a user. However, since it requires a token, I expect it to be able to extract the `operatorGuid`. Does it make sense?

Reply:
Actually, I don't think `SendEmailRoute` is an authenticated route. This means we're not going to have a `DDPAuth` token nor an operatorGuid. In these cases, I think we should just use `participantGuid` as operator guid.

Reply:
Yeah, makes sense, changed this.